shared/src/commonMain/kotlin/io.newm.shared/repositories/CardanoWalletRepository.kt

```diff
@@ -33,15 +34,10 @@ fun List<LedgerAssetMetadata>.getMusicMetadataVersion(): Int? {
 }
 
 data class NFTTrack(
-    val name: String,
-    val image: String,
-    val imageMimeType: String,
-    val files: List<File>,
-    val musicMetadataVersion: Int,
-    val releaseType: String?,
-    val releaseTitle: String?,
-    val distributor: String?,
-    val publicationDate: String?
+    val songName: String,
+    val imageUrl: String,
+    val songSrc: String,
+    val artist: List<String> = emptyList(),
```

Comment:
```suggestion
    val artists: List<String> = emptyList(),
```

Reply:
Are these the artist's display names or identifiers?  Thinking maybe they should be identifiers, since (eventually) we'll want to be able to let the user tap the artist name and open the artist page, like spotify.